src/main/java/com/ita/if103java/ims/controller/AccountController.java

```diff
@@ -1,16 +1,8 @@
 package com.ita.if103java.ims.controller;
 
 import com.ita.if103java.ims.dto.AccountDto;
-import com.ita.if103java.ims.dto.UserDto;
-import com.ita.if103java.ims.dto.transfer.ExistData;
-import com.ita.if103java.ims.dto.transfer.NewData;
-import com.ita.if103java.ims.entity.User;
 import com.ita.if103java.ims.service.AccountService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
-import org.springframework.security.core.annotation.AuthenticationPrincipal;
-import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
```

Comment:
Please do not use wildcards in imports
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports